### PR TITLE
Fixed login button ID selector 

### PR DESCRIPTION
**Pull request by LionZXY**
# Background

Now epic game store change id for login button from `login` to `sign-in`

# Actual state
- Auth through browser not work

# Expected state
- Auth through browser work

# Test plan
1. Run https://github.com/Revadike/epicgames-freebies-claimer and disable normal auth
2. Check that auth through browser work correctly

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -195,13 +195,16 @@ class EpicGamesClientLoginAdapter {
...
+    const login = credentials.login || credentials.email || credentials.username;
+    if (login && credentials.password) {
+      const loginWithEpicButton = await page.waitForSelector('#login-with-epic');
+      await loginWithEpicButton.click();
+      const usernameOrEmailField = await page.waitForSelector('#email');
+      await usernameOrEmailField.type(login, { delay: options.inputDelay });
+      const passwordField = await page.waitForSelector('#password');
+      await passwordField.type(credentials.password, { delay: options.inputDelay });
+      const loginButton = await page.waitForSelector('#sign-in:not(:disabled)');
+      await loginButton.click();
```
Comment (LionZXY) [inline index.js:207]: This is strange behaviour with merge conflict resolver

Comment (Revadike) [reply index.js:207]: merge conflict resolver?

Comment (LionZXY) [reply index.js:207]: Yup. This is sync with base repo

Comment (Revadike) [inline index.js:207]: Why is this changed?

**Comment by LionZXY** [comment]
After 10 hours of work, I still could not get epicgames-claimer to work. I won't do it anymore, I can only advise to improve support, set up auto-updating forks or merge changes into them. If your goal is really to improve the open-source community, consider it. 

Software in its current state is hard to maintain and use.
